test/pjrt/test_experimental_pjrt_tpu.py

```diff
@@ -144,6 +144,42 @@ def test_default_xla_devices(self):
         devices,
         [torch.device(f'xla:{i}') for i in range(expected_num_devices)])
 
+  @parameterized.named_parameters(('xla_model', xm.get_ordinal),
+                                  ('pjrt', pjrt.global_ordinal))
```

Comment:
I guess functionally `xm.get_ordinal` just calls `pjrt.global_ordinal` so they are the same?

Reply:
Right. This is confirming we get the same results from both endpoints in case the implementation changes (on purpose or by accident) in the future.